atlasdb-api/src/main/java/com/palantir/atlasdb/transaction/api/TransactionManager.java

```diff
@@ -320,10 +319,12 @@ default boolean isInitialized() {
      */
     TimelockService getTimelockService();
 
-    @DoNotDelegate
-    default LockWatchManager getLockWatchManager() {
-        return NoOpInternalLockWatchManager.INSTANCE;
-    }
+    /**
+     * Returns the lock watch manager used by this transaction manager.
+     *
+     * @return the lock watch manager service for this transaction manager
+     */
+    LockWatchManager getLockWatchManager();
```

Comment:
this is just a binary breaking change - will affect a number of consumers internally